extensions/wikia/Chat2/js/views/views.js

```diff
@@ -553,7 +551,9 @@ var NodeChatUsers = Backbone.View.extend({
 		if (actions.regular && actions.regular.length) {
 			var regularActions = ul.clone().addClass('regular-actions');
 			for (var i in actions.regular) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Restrict what this loop acts on by testing each property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AForIn)